source/renderer/app/stores/HardwareWalletsStore.ts

```diff
@@ -253,10 +235,16 @@ export default class HardwareWalletsStore extends Store {
   activeVotingWalletId: string | null | undefined = null;
   @observable
   votingData: VotingDataType | null | undefined = null;
-  // @ts-ignore ts-migrate(2304) FIXME: Cannot find name 'IntervalID'.
-  cardanoAdaAppPollingInterval: IntervalID | null | undefined = null;
+  cardanoAdaAppPollingInterval: {
```

Comment:
Should we rename this? I would expect the `intervalID` under this name. Perhaps `cardanoAdaAppPoller`?

Reply:
Also, can we extract the type declaration and just refer to it here?